rcl/test/rcl/test_arguments.cpp

```diff
@@ -317,8 +318,16 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_bad_alloc_unpar
   EXPECT_EQ(
     RCL_RET_BAD_ALLOC, rcl_arguments_get_unparsed(&parsed_args, bad_alloc, &actual_unparsed));
   rcl_reset_error();
+
   EXPECT_EQ(
-    RCL_RET_BAD_ALLOC, rcl_arguments_get_unparsed_ros(&parsed_args, bad_alloc, &actual_unparsed));
+    RCL_RET_INVALID_ARGUMENT,
+    rcl_arguments_get_unparsed_ros(nullptr, allocator, &actual_unparsed));
+  rcl_reset_error();
+
+  rcl_arguments_t empty_parsed_args = rcl_get_zero_initialized_arguments();
+  EXPECT_EQ(
+    RCL_RET_INVALID_ARGUMENT,
+    rcl_arguments_get_unparsed(&empty_parsed_args, allocator, &actual_unparsed));
```

Comment:
@Blast545 hmm these tests do not seem to correlate with the test case name? Maybe relocate?

Reply:
Done 2d43cd8